packages/portal-proto/src/features/GenomicTables/GenesTable/utils.tsx

```diff
@@ -69,7 +70,8 @@ export const useGenerateGenesTableColumns = ({
             classNames={{
               input: "checked:bg-accent checked:border-accent",
             }}
-            aria-label="checkbox for selecting table row"
+            aria-label={`checkbox for selecting genes table row ${row.id}`}
```

Comment:
consider using aria-labelledby instead, row.id is also not helpful as a label consider using something like symbol. Also your wording reads as 'checkbox for selecting genes table row ${row.id} checkbox not checked' this should be simplified. 'checkbox for selecting genes table row' will be read on every row and does not tell the user anything new

Reply:
using symbol now within the aria-label & shortened wording to be less repetitive.